.vscode/launch.json

```diff
@@ -31,6 +31,19 @@
                 "order": 1
             }
         },
+        {
+            "name": "Extension (web)",

```

Comment:
This is how you debug the web extension. It requires you run the 'Compile Web Extension' task first (which I talked about in the tasks.json)